packages/example-app-base/src/utilities/jumpToAnchor.ts

```diff
@@ -0,0 +1,29 @@
+const SCROLL_DISTANCE = 52;
+
+export function jumpToAnchor(anchor?: string, scrollDistance: number = SCROLL_DISTANCE): void {
+  const hash = anchor || _extractAnchorLink(window.location.hash);
```

Comment:
If the browser is _not_ looking at a `#` in the location, `hash` comes back as the full location and the rest of the function likely does not do what the caller intends. Should ensure that the anchor is actually active.

Reply:
Added logic to return undefined if there is no # in the URL